pkg/devspace/helm/v3/client.go

```diff
@@ -56,12 +65,11 @@ func (c *client) InstallChart(ctx devspacecontext.Context, releaseName string, r
 	// Chart settings
 	chartName := ""
 	if helmConfig.Chart.Source != nil {
-		chartName, err = dependencyutil.DownloadDependency(ctx.Context(), ctx.WorkingDir(), helmConfig.Chart.Source, ctx.Log())
+		chartName, err = c.DownloadChart(ctx, helmConfig)
```

Comment:
This results in two downloads when there are changes. I'm not sure what side effects of modifying the helm config name after the first DownloadChart call would have, so opting for less changes.